coderd/notifications/types/labels.go

```diff
@@ -0,0 +1,72 @@
+package types
+
+import (
+	"fmt"
+)
+
+// Labels represents the metadata defined in a notification message, which will be used to augment the notification
+// display and delivery.
+type Labels map[string]string
+
+func (l Labels) GetStrict(k string) (string, bool) {
```

Comment:
nit: `GetOk`

Reply:
Not loving either name, TBH.
Maybe `TryGet`? `GetSafe`? `GetIfExists`?

Reply:
Only reason I suggested `GetOk` is due to having seen similar verbiage elsewhere: https://pkg.go.dev/github.com/hashicorp/terraform/helper/schema#ResourceData.GetOk

Feel free to ignore this, it's just a nit and non-blocking.

Reply:
OK, prior art does make me feel a bit better about `GetOk` 👍 
Thanks